### DISCUSSION TITLE
option to remove "dont't have an account? sign up"

### DISCUSSION THREAD

**Discussion by jedierikb** [supabase/supabase#18199] [Feature Requests]
Is there a way in auth-ui to **not** show the link for "_dont't have an account? sign up_"?

I am building a site where I want a select few people to be able to sign in, but do not want to allow new accounts.

If this is not an option, what is the recommended way to enable this feature?

**Comment by Hallidayo** [comment]
Hi @jedierikb, Their is a `showLinks` prop on `<Auth />` which can be used which hides all of the links.
